app/controllers/v3/service_instances_controller.rb

```diff
@@ -297,6 +311,13 @@ def update_managed(service_instance)
     action = V3::ServiceInstanceUpdateManaged.new(service_instance, message, user_audit_info, message.audit_hash)
     action.preflight!
     if action.update_broker_needed?
+      logger.info(
+        "Updating managed service instance with name '#{service_instance.name}' " \
+        "using service plan '#{service_instance.service_plan.name}' " \
+        "from service offering '#{service_instance.service_plan.service.label}' " \
+        "provided by broker '#{service_instance.service_plan.service.service_broker.name}'."
```

Comment:
When I tested this, a database query to fetch broker information was executed.

It seems as if `service_instance.reload` ([here](https://github.com/cloudfoundry/cloud_controller_ng/blob/511e64b9d933de1fe9454e2ccc5108e4a0e9b6a7/app/actions/v3/service_instance_update_managed.rb#L272)) removes the associated objects (or at least some of them).

So for the `update_managed` part, eager loading does not have the expected benefits.

Additional question: can the service plan be updated? This would even make the eager loading completely obsolete in such a case.